### PR TITLE
PHP7+ / recent MySQL compatibility

### PR DESCRIPTION
**Pull request by vendethiel**
Doesn't use the latest PHP or MySQL but rather those I have installed, that is:

MySQL 5.7.26
PHP 7.3.6

### CODE REVIEW

**Review by ProTogen11 — APPROVED** [review]

File: templates/default/default.cfg [hunk 1]
```diff
@@ -716,15 +716,15 @@ $template->assign_vars(array(
...
+	'TEMPLATE_COPYRIGHT_LINK' => ( isset($template_copyright_link) ? $template_copyright_link : '' ),
 	'SHOW_HIDE_PADDING' => $show_hide_padding,
 
 	'IMG_THL' => $images['tbl_h_l'],
 	'IMG_THC' => $images['tbl_h_c'],
 	'IMG_THR' => $images['tbl_h_r'],
-	'IMG_TBL' => $images['tbl_b_l'],
-	'IMG_TBC' => $images['tbl_b_c'],
-	'IMG_TBR' => $images['tbl_b_r'],
+	'IMG_TBL' => ( isset($images['tbl_b_l']) ? $images['tbl_b_l'] : '' ),
```
Comment (vendethiel) [inline templates/default/default.cfg:725]: Not sure why these aren't defined

File: templates/default/default.cfg [hunk 2]
```diff
@@ -754,8 +754,8 @@ $template->assign_vars(array(
 	'IMG_ARROW_DOWN' => '<img src="' . $images['arrow_down'] . '" alt="down" title="down" />',
 	'IMG_ARROW_LEFT' => '<img src="' . $images['arrow_left'] . '" alt="left" title="left" />',
 	'IMG_MENU_SEP' => $images['menu_sep'],
-	'IMG_POST_PREV' => $images['icon_post_prev'],
-	'IMG_POST_NEXT' => $images['icon_post_next'],
+	'IMG_POST_PREV' => ( isset($images['icon_post_prev']) ? $images['icon_post_prev'] : '' ),
```
Comment (vendethiel) [inline templates/default/default.cfg:757]: Same here

**Comment by MightyGorgon** [comment]
Hi, thank you for the suggestion, I will merge it hopefully next week, I'm currently out until next week.

File: adm/admin_account.php [hunk 3]
```diff
@@ -8,15 +8,14 @@
 *
 */
 
-if (!defined('IN_ICYPHOENIX')) define('IN_ICYPHOENIX', true);
-
-if(!empty($setmodules))
+if(defined('IN_ICYPHOENIX') && !empty($setmodules))
```
Comment (MightyGorgon) [inline adm/admin_account.php:11]: I will accept this, but at the same time I will need to understand which is the advantage to have it in this form... I may revert it if I'm not able to double check all single pages.

**Comment by MightyGorgon** [comment]
I have quickly reviewed and almost everything seems to be fine. I just need to double check few things when I have some time. So I will currently merge everything and eventually revert back what I'm not convincend of. :-)

Thanks for your help.

**Pull request merged**